### ISSUE TITLE
Provide a way to allow fractional seconds rather than milliseconds

### ISSUE THREAD

**Issue by mungojam** [Humanizr/Humanizer#695]
For example, `TimeSpan.FromMilliseconds(500).Humanize(precision: 2, allowFractionalSeconds: true)` would return:

> 0.5 seconds

while `(TimeSpan.FromDays(2) + TimeSpan.FromMilliseconds(700)).Humanize(precision: 2, useFractionalSeconds: true)`  would return say:

> 2 days, 1 second

Not bothered by the syntax, but there doesn't seem to be a way to do this at present. I think our end users would be more comfortable seeing fractional seconds than milliseconds but I also want to allow for the possibility of times in hours and minutes.

**Comment by kcragin** [comment]
I totally agree with this. It is a bit awkward to read **X seconds, Y milliseconds**, and humans tend not to say it that way.

**Comment by clairernovotny** [comment]
Fixed on `main` by #1885 and planned for the next v4 release.

Thanks @mungojam for reporting the fractional-seconds use case, and @kcragin for contributing to the discussion. The new distinct APIs preserve existing `TimeSpan.Humanize` behavior while supporting localized fractional terminal seconds.
